### DISCUSSION TITLE
R2DBC LocalTime Support

### DISCUSSION THREAD

**Discussion by bsamartins** [micronaut-projects/micronaut-data#1168] [Q&A] [answered]
Hi

I'm using Postgres and `micronaut-data-rdbc`.
In particular table on of the columns is of type `time` (time without timestamp), mapped on my entity as `LocalTime`.
```
@MappedProperty(value = "my_time_column")
val myTimeColumn: LocalTime? 
```

When I attempt to insert it, it fails mentioning that type should be of Timestamp and I should create a converter.

The issue with the converter is that it introduces date and timezone / offset in the conversion.

Is there a way I can change my `@MappedProperty` to support it?
Does R2DBC support LocalTime?

**Comment by dstepanov** [comment, marked as answer]
I have created a PR that supports storing `LocalTime` using `Timestamp`.
At the moment we don't have special database mapping for time, everything is being mapped thru timestamp. 

https://github.com/micronaut-projects/micronaut-data/pull/1170

**Reply by bsamartins** [reply]
Thanks for clarifying.
I`ll try the converter approach now.
